src/Utils/TypeInfo.php

```diff
@@ -363,52 +363,42 @@ static function ($arg) use ($node) {
                     $inputFieldType = $inputField ? $inputField->getType() : null;
                 }
                 $this->defaultValueStack[] = $inputField && $inputField->defaultValueExists() ? $inputField->defaultValue : Utils::undefined();
-                $this->inputTypeStack[]    = Type::isInputType($inputFieldType) ? $inputFieldType : null;
+                $this->inputTypeStack[]    = $inputFieldType;
```

Comment:
Same here

Reply:
Sure. We can revisit some other time. reverted in 78295d7